pkgs/development/node-packages/overrides.nix

```
@@ -274,6 +274,9 @@ final: prev: {
     };
   };
 
+  protoc-gen-ts =
+    final."@protobuf-ts/plugin".override { name = "protoc-gen-ts"; };
```

style nit (for consistency with the rest of the file):

```suggestion
  protoc-gen-ts = final."@protobuf-ts/plugin".override {
    name = "protoc-gen-ts";
  };
```

---

also, s/final/prev?

---

```suggestion
  protoc-gen-ts = prev."@protobuf-ts/plugin".override {
    name = "protoc-gen-ts";
  };
```